.gitignore

```diff
@@ -12,3 +12,5 @@
 /test/dict.bin
 /xcode
 /node_modules
+/dist
```

Comment:
Can you please also update `.npmignore`?

Reply:
Done fdab98a